public/app/plugins/datasource/grafana-azure-monitor-datasource/azure_monitor/azure_monitor_datasource.ts

```diff
@@ -105,12 +105,10 @@ export default class AzureMonitorDatasource extends DataSourceWithBackend<AzureM
     const dimensionFilters = (item.dimensionFilters ?? [])
       .filter((f) => f.dimension && f.dimension !== 'None')
       .map((f) => {
-        const filter = templateSrv.replace(f.filter ?? '', scopedVars);
```

Comment:
can you double check if you can remove this filter? my question is because I don't know if queries that reach this function has been migrated already or not. I believe they will but better to be sure.

Reply:
A query that gets to this point will be migrated first based on hitting [this line](https://github.com/grafana/grafana/blob/7b9929fffe88d81dd0a5dd1397bbd85e88fb9fe0/public/app/plugins/datasource/grafana-azure-monitor-datasource/datasource.ts#L73) and then [this line](https://github.com/grafana/grafana/blob/7b9929fffe88d81dd0a5dd1397bbd85e88fb9fe0/packages/grafana-runtime/src/utils/DataSourceWithBackend.ts#L143).